pypxe-server.py

```diff
@@ -117,7 +119,7 @@
                 args.NETBOOT_FILE = 'boot.http.ipxe'
 
         #serve all files from one directory
-        os.chdir (args.NETBOOT_DIR)
+        os.chdir(args.NETBOOT_DIR)
```

Comment:
humm.. no, bcause the module has `self.netbootDirectory = serverSettings.get('netbootDirectory', '.')`